ftplugin/php/mappings.vim

```diff
@@ -4,19 +4,21 @@
 " Phpactor does not assume any mappings automatically, the following mappings
 " are available for you to copy: >
 "
-"   nmap <Leader>u :PhpactorImportClass<CR>
-"   nmap <Leader>ua :PhpactorImportAllClasses<CR>
-"   nmap <Leader>mm :PhpactorContextMenu()<CR>
-"   nmap <Leader>nn :PhpactorNavigate()<CR>
-"   nmap <Leader>oo :PhpactorGotoDefinition()<CR>
-"   nmap <Leader>oh :PhpactorGotoDefinitionHsplit()<CR>
-"   nmap <Leader>ov :PhpactorGotoDefinitionVsplit()<CR>
-"   nmap <Leader>ot :PhpactorGotoDefinitionTab()<CR>
-"   nmap <Leader>K :PhpactorHover()<CR>
-"   nmap <Leader>tt :PhpactorTransform()<CR>
-"   nmap <Leader>cc :PhpactorClassNew()<CR>
-"   nmap <silent><Leader>ee :PhpactorExtractExpression(v:false)<CR>
-"   vmap <silent><Leader>ee :<C-U>PhpactorExtractExpression(v:true)<CR>
-"   vmap <silent><Leader>em :<C-U>PhpactorExtractMethod()<CR>
+"   augroup PhpactorMappings
+"     au!
+"     au FileType php nmap <Leader>u :PhpactorImportClass<CR>
+"     au FileType php nmap <Leader>mm :PhpactorContextMenu<CR>
+"     au FileType php nmap <Leader>nn :PhpactorNavigate<CR>
+"     au FileType php nmap <Leader>oo :PhpactorGotoDefinition<CR>
+"     au FileType php nmap <Leader>oh :PhpactorGotoDefinitionHsplit<CR>
+"     au FileType php nmap <Leader>ov :PhpactorGotoDefinitionVsplit<CR>
+"     au FileType php nmap <Leader>ot :PhpactorGotoDefinitionTab<CR>
+"     au FileType php nmap <Leader>K :PhpactorHover<CR>
+"     au FileType php nmap <Leader>tt :PhpactorTransform<CR>
+"     au FileType php nmap <Leader>cc :PhpactorClassNew<CR>
+"     au FileType php nmap <silent><Leader>ee :PhpactorExtractExpression<CR>
+"     au FileType php vmap <silent><Leader>ee :<C-u>PhpactorExtractExpression<CR>
+"     au FileType php vmap <silent><Leader>em :<C-u>PhpactorExtractMethod<CR>
+"   augroup END
```

Comment:
The autocomand with PHP filetype are not useful in this context since you already are in an `ftplugin` that will be loaded only for PHP.
But I would instead add `<buffer`> so that the mappings are only defined for the current buffer, that will prevent any overlap with mappings from a plugin for another filetype.

Reply:
Added `<buffer>` mapping